internal/xds/translator/listener_test.go

```diff
@@ -101,3 +103,35 @@ func Test_buildTCPProxyHashPolicy(t *testing.T) {
 		})
 	}
 }
+
+func Test_buildGenerateRequestID(t *testing.T) {
```

Comment:
Can you also add a test case using testdata for xds translator_test?

Reply:
Sure, added